### DISCUSSION TITLE
How to report a vercel.app site?

### DISCUSSION THREAD

**Discussion by llave2** [vercel/community#840] [Help] [answered]
Hello, this page https://megatelevisionhd.com/en-vivo/telemundo/ is using vercel.app iframes to get multiple vercel.app subdomains in order to avoid blocking from streaming websites and be able to use their streams in his websites, everytime the streaming platforms block their vercel.app subdomain, he just create another subdomain and put its html webplayer and iframe it to their website and keep using the streaming links, that should be violation of TOS
right now the subdomain being used is https://fadwd.vercel.app/ and another is https://zkfpda.vercel.app/ , I don't find anywhere to report this on the vercel webpage as I found in other platforms and reported it before and they deleted their accounts

**Comment by swarnava** [comment, marked as answer]
Hi,

You can email at abuse@vercel[dot]com
